packages/lib-user/src/components/GroupStats/helpers/getHeaderItems.spec.js

```diff
@@ -1,5 +1,7 @@
 import getHeaderItems from './getHeaderItems'
 
+const DEFAULT_HANDLER = (key) => key
```

Comment:
`t` from `useTranslation()` cannot be called outside of a functional component, so this test suite must be passed a default function each time.